src/components/molecules/definitionitem/DefinitionItem.scss

```diff
@@ -7,10 +7,33 @@
 }
 
 .term-title {
-	display: inline-block;
 	margin-bottom: 10px;
 }
 .definition-link {
 	margin-left: 16px;
 	vertical-align: super;
 }
+
```

Comment:
Have we confirmed with design the hover and focus states (keyboard focus) for this button?